entries/h/hide.me.json

```
@@ -1,13 +1,11 @@
 {
   "hide.me VPN": {
     "domain": "hide.me",
-    "contact": {
-      "email": "support@hide.me",
-      "facebook": "www.hide.me",
-      "twitter": "thevpncompany"
-    },
+    "tfa": [
+      "totp"
+    ],
```

As long as the official documentation is not up to date, this blog post will do:
```suggestion
    ],
    "documentation": "https://hide.me/en/blog/hide-me-vpn-now-supports-2fa/",
```